filter.js

```diff
@@ -112,45 +117,25 @@ Filter.prototype.processAndCacheFile =
         e.file = relativePath;
         e.treeDir = srcDir;
         throw e;
-      });
-
-  function hash(filePaths) {
-    return filePaths.map(function hashSubTree(filePath) {
-      return helpers.hashTree(srcDir + '/' + filePath);
-    }).join(',');
-  }
+      })
+
+  function copyToCache() {
+    var entry = {
+      hash: hash(srcDir, relativePath),
+      inputFile: relativePath,
+      outputFile: destDir + '/' + internalGetDestFilePath(self, relativePath),
+      cacheFile: self.cachePath + '/' + relativePath
+    };
 
-  function symlinkOrCopyFromCache(cacheEntry) {
-    var outputFiles = cacheEntry.outputFiles;
-    var cacheFiles = cacheEntry.cacheFiles;
-    for (var i = 0, ii = outputFiles.length; i < ii; ++i) {
-      var dest = destDir + '/' + outputFiles[i];
-      mkdirp.sync(path.dirname(dest));
-      symlinkOrCopySync(self.cachePath + '/' + cacheFiles[i], dest);
+    if (fs.existsSync(entry.cacheFile)) {
+      fs.unlinkSync(entry.cacheFile);
```

Comment:
This line never seems to get hit --- I'm guessing because of the `deleteExcept()` method. I'm also not seeing any lines from coverage.js in coveralls, so it's hard to see if `deleteExcept()` is the cause of this not getting hit or not.

If the line isn't used, lets get rid of it, and if it is used, lets make sure coverage reports make this clear